client/src/components/organisms/LogoModal.tsx

```diff
@@ -12,7 +12,7 @@ interface Props {
 
 const LogoModal: FunctionComponent<Props> = (props: Props) => {
   return (
-    <CommonModal class="modal" size="default" show={props.show} handleClose={props.handleClose}>
+    <CommonModal class="logo-modal" size="default" show={props.show} handleClose={props.handleClose}>
```

Comment:
Classname "modal" conflicts with bootstrap's classname for modals (which is also "modal")